index.dd

```diff
@@ -76,6 +76,22 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE
+----
+// Compile-Time Sort in D
+void main()
+{
+    import std.algorithm, std.stdio;
+    writeln("Starting program.");
+    // Constant declaration
```

Comment:
What about adding some blank lines to make the code "breathe" better? That, or move the comments to EOL comments instead of occupying a single line each?